packages/react/src/components/Tile/Tile.js

```diff
@@ -574,20 +605,20 @@ export class ExpandableTile extends Component {
       tileMaxHeight, // eslint-disable-line
       tilePadding, // eslint-disable-line
       handleClick, // eslint-disable-line
-      onClick,
-      onKeyUp,
+      onClick, // eslint-disable-line
```

Comment:
Probably worth adding a note why its disabled so we can change the rule or fix it in the future